README.md

```diff
@@ -14,7 +14,7 @@ if the cost exceeds the limit.
   that use their own bundler or small npm libraries with many files.
 * Can calculate **the time** it would take a browser
   to download and **execute** your JS. Time is a much more accurate
-  and understandable metric compared to the size in bytes.
+  and understandable metric compared to the size in bytes. Additionally, you can customize it via config for every check with network speed, latency and so on.
```

Comment:
Looks good but we need more examples in docs

Reply:
Thank you for the suggestion. I've added the documentation to the time plugin and left the reference in the root readme.